server/app.py

```diff
@@ -3,9 +3,22 @@
 
 from fastapi import FastAPI, File, UploadFile
 from handlers import calendar_handler
+from fastapi.middleware.cors import CORSMiddleware
 
 app = FastAPI()
 
+origins = [
+    "http://localhost:3000",
+    "https://localhost:3000",
+]
+
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins=origins,
+    allow_credentials=True,
+    allow_methods=["*"],
+    allow_headers=["*"],
+)
```

Comment:
Need to format using black. I recommend using the `ms-python.black-formatter` extension or run `black .` inside of `server`